### PR TITLE
strip header row from uploads

### LINKED ISSUE #153: strip headers on csv data uploads

### PR DESCRIPTION
**Pull request by dduugg**
resolves #153

### CODE REVIEW

File: server/utils/buffer.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  indexOf: function(buffer, charCode) {
```
Comment (dduugg) [inline server/utils/buffer.js]: astoundingly, buffers have a `length` property and can be indexed into, but lack an `indexOf` method: http://nodejs.org/api/buffer.html

File: server/config/middleware.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 var morgan = require("morgan");
 var bodyParser = require("body-parser");
-var helpers = require("./helpers.js");
+var helpers = require("./helpers");
```
Comment (zindlerb) [inline server/config/middleware.js]: :ocean: :sunglasses: :ocean:

**Comment by cmperez** [comment]
drunk code looks good :+1:

**Comment by zindlerb** [comment]
:beer: :beers: :beer: :+1:

**Pull request merged**